README.md

```diff
@@ -18,4 +18,14 @@ hugo-golunr
 Pretty easy, huh? After running `hugo-golunr`, you'll see a `search_index.json` file in your
 `./static` directory. Just load that in your theme.
 
+## Tidying
+I have moved the code to a standard format for a command line format and added a mod.ls
```

Comment:
Mh, I disagree with "standard format" - the standard in go is "make it as simple as it can be", which this format isn't. I don't like hiding things in `internal` packages, since they cannot be re-used. 
It's a matter of preference for sure, but I tend to disagree with this change.

Also, this should live in a commit message, not in the README 🙂

Reply:
Thanks for feedback  I think I need to keep the yours and my versions separate.  I will try and create a better update for yours.